packages/frontend/src/admin-pages/components/nonprofit/yourPage/tabs/Media.vue

```diff
@@ -46,7 +46,7 @@
           :to="{ name: 'nonprofit-your-page-media-videos-add' }"
           role="button"
           class="c-btn c-btn--sm c-btn--icon"
-          :disabled="disableAddButton"
+          :is-active="!disableAddButton"
```

Comment:
The `is-active` attribute seems to be having no effect.
`:disabled="disableAddButton ? true : null"` will add the disabled attribute (when true) and will omit it otherwise (so not have the issue `disabled="false"` was causing).